lib/sisimai/bite/email/ezweb.rb

```diff
@@ -0,0 +1,260 @@
+module Sisimai::Bite::Email
+  module EZweb
+    # Sisimai::Bite::Email::EZweb parses a bounce email which created by au EZweb.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/EZweb.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'from'       => %r/[<]?(?>postmaster[@]ezweb[.]ne[.]jp)[>]?/i,
+        :'subject'    => %r/\AMail System Error - Returned Mail\z/,
+        :'received'   => %r/\Afrom[ ](?:.+[.])?ezweb[.]ne[.]jp[ ]/,
+        :'message-id' => %r/[@].+[.]ezweb[.]ne[.]jp[>]\z/,
+      }
+      Re1 = {
+        :begin  => %r{\A(?:
```

Comment:
__[rubocop]__
* Layout/SpaceAroundOperators: Operator `=>` should be surrounded by a single space. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->